romeo_dcm:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Controllers ------------------------------------------------
  Head_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - NeckYaw
      - NeckPitch
      - HeadPitch
      - HeadRoll
  Torso_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - TrunkYaw
#      - waist
  LeftArm_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - LShoulderPitch
      - LShoulderYaw
      - LElbowRoll
      - LElbowYaw
      - LWristRoll
      - LWristYaw
  RightArm_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - RShoulderPitch
      - RShoulderYaw
      - RElbowRoll
      - RElbowYaw
      - RWristRoll
      - RWristYaw
  LeftHand_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - LWristPitch
      - LHand
  RightHand_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - RWristPitch     
      - RHand
  LeftLeg_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - LHipYaw
      - LHipRoll
      - LHipPitch
      - LKneePitch
      - LAnklePitch
      - LAnkleRoll
#      - LToePitch
  RightLeg_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - RHipYaw
      - RHipRoll
      - RHipPitch
      - RKneePitch
      - RAnklePitch
      - RAnkleRoll
#      - RToePitch
